### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by toolCHAINZ** [prove-rs/z3.rs#294]
I noticed it's been a while since a new version of these bindings have been pushed to crates.io and there's some really good stuff missing from that version.

Any possibility of a release in the near future?

**Comment by waywardmonkeys** [comment]
I'd like to ... I want to get the bundled Z3 updated, but that is still failing and I need to figure out the right fix. Just now in fixing something else, the WebAssembly build failed due to an error in Z3 and I've just submitted a PR for that. (That will require a new release of Z3...)

Would be good to do though!

**Comment by toolCHAINZ** [comment]
Gotcha, thanks! Please let me know if there's any way I can help out; I've loved using this library and wouldn't mind contributing if there was something I could do. Out of curiousity, what is failing? I just pulled down `z3.rs` and bumped `z3-sys/z3` to `z3-4.13.0` and all the `z3-sys` and `z3` tests passed (ARM macOS).

**Comment by toolCHAINZ** [comment]
Poking around at this a bit on my Windows 11 machine (apologies if you've already triaged all this):

* `cargo build --features bundled` succeeds
* `cargo test --features bundled` fails
* `cargo test --release --features bundled` succeeds

It looks like for some reason the debug build is emitting a linker flag to link to `msvcrt.lib`, which is the release version of the library. But there seems to be a default link to the debug version of the same library?

```
  = note: LINK : warning LNK4098: defaultlib 'MSVCRTD' conflicts with use of other libs; use /NODEFAULTLIB:library
```

**Comment by waywardmonkeys** [comment]
I know that it is from doing debug builds on Windows, but not much beyond that. I haven't been booting my Windows machine often as of late ... I also know that it started when I increased the cmake version requirements in upstream Z3 (and therefore being subject to changes in the cmake policies).

**Comment by toolCHAINZ** [comment]
Ok, I just tracked down some `cmake` stuff. Bisected `cmake` versions and it seems that `3.15` is the first version with broken debug tests.

Still looking through the changelog for that version, but the following item does pop out:


> The [CMAKE_MSVC_RUNTIME_LIBRARY](https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html#variable:CMAKE_MSVC_RUNTIME_LIBRARY) variable and [MSVC_RUNTIME_LIBRARY](https://cmake.org/cmake/help/latest/prop_tgt/MSVC_RUNTIME_LIBRARY.html#prop_tgt:MSVC_RUNTIME_LIBRARY) target property were introduced to select the runtime library used by compilers targeting the MSVC ABI. See policy [CMP0091](https://cmake.org/cmake/help/latest/policy/CMP0091.html#policy:CMP0091).

**Comment by toolCHAINZ** [comment]
Not at all a CMake expert here, but I'm wondering if this is a `cmake-rs` problem

This snippet seems to indicate that `cmake-rs` is attempting to communicate `/MT` and `/MD` type flags through `CMAKE_{LANG}_FLAGS_{CONFIG}` variables:

https://github.com/rust-lang/cmake-rs/blob/c4a60dd154dd90e469dffc41a1faa717704f90b3/src/lib.rs#L731-L754

But [CMP0091](https://cmake.org/cmake/help/latest/policy/CMP0091.html#policy:CMP0091) seems to indicate that MSVC lib selection is no longer done through these flags. So maybe `cmake-rs` is using a deprecated flag, cmake is ignoring it, and that's causing the mismatch.

**Comment by toolCHAINZ** [comment]
I've made https://github.com/prove-rs/z3.rs/pull/295 which contains a change that fixes debug builds on my machine.

**Comment by toolCHAINZ** [comment]
Looks like there's finally a new z3 release! 

https://github.com/Z3Prover/z3/releases/tag/z3-4.13.2

**Comment by rayzhuca** [comment]
hi @waywardmonkeys ! giving a gentle bump on this. waiting for the new version which contains a fix for #190. would be great if this is pushed soon assuming theres no blockage

**Comment by waywardmonkeys** [comment]
Yes, we should!

There're a new PR that I'd like to get in and a couple of older ones that maybe should ... anyone have anything else they want to see get in this week?

**Comment by toolCHAINZ** [comment]
Nothing for me!

**Comment by toolCHAINZ** [comment]
Hey @waywardmonkeys! Gentle bump on this, any chance of a crates.io push in the next couple weeks?

**Comment by waywardmonkeys** [comment]
Yes! I was starting to look into it recently and then ran into some other things. I am traveling for the next 30 hours. Will look into it early in the coming week!

**Comment by toolCHAINZ** [comment]
Hi @waywardmonkeys, gentle bump on this. I hope you had a restful holiday!

**Comment by toolCHAINZ** [comment]
Hi @waywardmonkeys! Gentle bump on this.

**Comment by toolCHAINZ** [comment]
Hey @waywardmonkeys, sorry to spam! Any chance of a crates.io push soon? Thanks!

**Comment by toolCHAINZ** [comment]
Hi @waywardmonkeys! Just thought I'd pop in and check on this! Do you have any plans to push a release to crates.io in the near future (or the medium future :) )?

**Comment by toolCHAINZ** [comment]
Closing my own issue. 😆 

The latest `master` (as of today) has been published as `z3 0.13.0` and `z3-sys 0.9.0`
